### PR TITLE
[v2.4] change k3s-chart description

### PR DESCRIPTION
**Pull request by MonzElmasry**
issue : rancher/rke2#154

### CODE REVIEW

File: charts/rancher-k3s-upgrader/0.2.0/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 name: rancher-k3s-upgrader
-description: Enables a k3s cluster to update itself by reacting to Plan CRs.
-home: https://github.com/rancher/system-charts/charts/system-upgrade-controller
+description: Enables a k3s or rke2 cluster to update itself by reacting to Plan CRs.
+  Users do not need to manualy upgrade this app. It will be autaomtically upgraded to the latest version when upgrading a cluster
```
Comment (rmweir) [inline charts/rancher-k3s-upgrader/0.2.0/Chart.yaml]: autaomtically -> automatically

**Review by rmweir — CHANGES_REQUESTED** [review]

File: charts/rancher-k3s-upgrader/0.2.0/Chart.yaml [hunk 2]
```diff
@@ -1,8 +1,9 @@
 apiVersion: v1
 name: rancher-k3s-upgrader
-description: Enables a k3s cluster to update itself by reacting to Plan CRs.
-home: https://github.com/rancher/system-charts/charts/system-upgrade-controller
+description: Enables a k3s or rke2 cluster to update itself by reacting to Plan CRs.
+  Users do not need to manualy upgrade this app. It will be automtically upgraded to the latest version when upgrading a cluster
```
Comment (rmweir) [inline charts/rancher-k3s-upgrader/0.2.0/Chart.yaml]: manualy -> manually

**Review by rmweir — CHANGES_REQUESTED** [review]

**Review by rmweir — APPROVED** [review]
LGTM

**Review by galal-hussein — COMMENTED** [review]
LGTM

**Review by galal-hussein — APPROVED** [review]

**Pull request merged**